src/dso_api/dynamic_api/fields.py

```diff
@@ -91,48 +89,14 @@ def to_representation(self, value):
         return f"{value}?{sas_token}"
 
 
-class LooseRelationUrlField(serializers.CharField):
```

Comment:
Ive renamed and repurposed this class to just serialize the url based on the pk value passed to it